substrate/primitives/arithmetic/src/per_things.rs

```diff
@@ -496,16 +496,35 @@ where
 	(x / maximum) * part_n + c
 }
 
+/// Checked compute of the error due to integer division in the expression `x / denom * numer`.
```

Comment:
```suggestion
/// Unchecked computation of the error due to integer division in the expression `x / denom * numer`.
```
And please mention the checked version. We should generally used the checked versions. One followup task would be to replace the usage of the unchecked things.